src/store/getters/state-getters.ts

```diff
@@ -130,6 +130,16 @@ export const getBusinessContact = (state: StateIF): BusinessContactIF => {
   return getDefineCompanyStep(state).businessContact
 }
 
+/** The Rules object. */
+export const getRules = (state: StateIF): any => {
+  return {} // *** FUTURE
```

Comment:
For whomever implements the Rules step (and below, the Memorandum step).

NB: This project now has a Rules interface that is not the same as this Rules data. At least one of these should be renamed... left to a future developer :)